platform/broadcom/rules.dep

```diff
@@ -0,0 +1,25 @@
+#DPKG FRK
+include $(PLATFORM_PATH)/sai-modules.dep
+include $(PLATFORM_PATH)/sai.dep
+include $(PLATFORM_PATH)/platform-modules-s6000.dep
+include $(PLATFORM_PATH)/platform-modules-dell.dep
+include $(PLATFORM_PATH)/platform-modules-arista.dep
+include $(PLATFORM_PATH)/platform-modules-ingrasys.dep
+include $(PLATFORM_PATH)/platform-modules-accton.dep
+include $(PLATFORM_PATH)/platform-modules-alphanetworks.dep
+include $(PLATFORM_PATH)/platform-modules-inventec.dep
+include $(PLATFORM_PATH)/platform-modules-cel.dep
+include $(PLATFORM_PATH)/platform-modules-delta.dep
+include $(PLATFORM_PATH)/platform-modules-quanta.dep
+#include $(PLATFORM_PATH)/platform-modules-mitac.dep
+include $(PLATFORM_PATH)/platform-modules-juniper.dep
+include $(PLATFORM_PATH)/platform-modules-brcm-xlr-gts.dep
+include $(PLATFORM_PATH)/docker-syncd-brcm.dep
+include $(PLATFORM_PATH)/docker-syncd-brcm-rpc.dep
+include $(PLATFORM_PATH)/docker-saiserver-brcm.dep
+include $(PLATFORM_PATH)/one-image.dep
+include $(PLATFORM_PATH)/one-pde-image.dep
+include $(PLATFORM_PATH)/raw-image.dep
+include $(PLATFORM_PATH)/one-aboot.dep
+include $(PLATFORM_PATH)/libsaithrift-dev.dep
+include $(PLATFORM_PATH)/docker-ptf-brcm.dep
```

Comment:
Is `$(SYNCD)_CACHE_MODE` missing in this PR? I can only see `$(SYNCD_VS)_CACHE_MODE`